resources/asciidoctor/spec/elastic_compat_tree_processor_spec.rb

```diff
@@ -96,4 +96,20 @@
       expect(actual).to eq(expected.strip)
     end
   end
+
+  it "doesn't mind when the snippet inside a definition list" do
+    actual = convert <<~ASCIIDOC
```

Comment:
I think I mentioned this in the last PR I reviewed, maybe eventually you can use `context` and `let` blocks instead in the specs.
For example, this case would have a context `when the snippet is inside a definition list` and then the `it` block would be `generates the correct string` (or whatever). The `actual` and `expected` variables would be `let` blocks.
But I think that change would be part of a larger rspec refactor and perhaps out of the scope of this PR.

[This](https://github.com/elastic/elasticsearch-ruby/blob/6.x/elasticsearch-api/spec/elasticsearch/api/utils_spec.rb#L89-L130) is an example of how context blocks and let blocks can be used.